house/views.py

```diff
@@ -46,29 +46,24 @@ def house_view(request):
     return render(request, MINE_MINE_PAGE_ROUTE, context)
 
 
-def load_cities(request):
-    country_id = request.GET.get('country')
-    cities = City.objects.filter(country_id=country_id).order_by('name')
-    context = {'cities': cities}
-    return render(request, GLOBAL_PAGE_CITY_DROPDOWN_ROUTE, context)
-
-
 @login_required
 def house_create(request):
     errorMsg = ""
     if request.method == 'POST':
         house_form = HouseCreationForm(request.POST)
         if house_form.is_valid():
             cleaned_data = house_form.cleaned_data
-            House.create_house(user=request.user,
-                               name=cleaned_data['name'],
-                               public=cleaned_data['public'],
-                               parent_profession_1=cleaned_data['parent_profession_1'],
-                               parent_profession_2=cleaned_data['parent_profession_2'],
-                               country=cleaned_data['country'],
-                               city=cleaned_data['city'],
-                               children=cleaned_data['children'],
-                               income=cleaned_data['income'])
+            House.create_house(
+                user=request.user,
+                name=cleaned_data['name'],
+                public=cleaned_data['public'],
+                parent_profession_1=cleaned_data['parent_profession_1'],
+                parent_profession_2=cleaned_data['parent_profession_2'],
+                country=cleaned_data['country'],
+                city=cleaned_data['city'],
+                children=cleaned_data['children'],
+                income=cleaned_data['income'],
+            )
```

Comment:
Is that relevant? isn't it just formatting
If so, all related formatting changes should be in a different PR.